projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-renderer.component.ts

```diff
@@ -124,7 +123,7 @@ export class TableWidgetRendererComponent
   public ngOnInit(): void {
     super.ngOnInit();
 
-    this.onModeChange(this.model.mode);
+    // This.onModeChange(this.model.mode);
```

Comment:
no longer needed? how does the initial view get set? if no longer needed, remove comment

Reply:
For this iteration, I don't have the different views in the URL, so not deep linkable. We just show the first view option. 

We need to decide how/if we want the different view toggles to be deep linkable (I think we do) and we need to decide if we want them as a route or a url param. I'm leaning towards param, but am open.